systemSmartContractsConfig.toml

```
@@ -2,24 +2,25 @@
     GenesisNodePrice = "2500000000000000000000" #2.5K eGLD
     MinStakeValue = "100000000000000000000" #100 eGLD
     MinUnstakeTokensValue = "10000000000000000000" #10eGLD
-    UnBondPeriod = 144000
-    UnBondPeriodInEpochs = 10
+    UnBondPeriod = 250
+    UnBondPeriodInEpochs = 1
     MinStepValue = "100000000000000000000"
     NumRoundsWithoutBleed = 100
     MaximumPercentageToBleed = 0.5
     BleedPercentagePerRound = 0.00001
-    MaxNumberOfNodesForStake = 2169
+    MaxNumberOfNodesForStake = 64
     UnJailValue = "2500000000000000000" #0.1% of genesis node price
     ActivateBLSPubKeyMessageVerification = true
     StakeLimitPercentage = 1.0 #fraction of value 1 - 100%, for the time being no stake limit
-    NodeLimitPercentage = 0.03125 #fraction of value 0.1 - 10% (1600*0.03125=50)
+    NodeLimitPercentage = 0.1 #fraction of value 0.1 - 10%
```

We have a [PR](https://github.com/multiversx/mx-chain-sovereign-go/pull/33) for this 
```suggestion
    NodeLimitPercentage = 0.4 #fraction of value 0.4 - 40%
```

---

Not needed anymore